libs/chlorophyll/scss/common/_focus.scss

```diff
@@ -30,10 +38,11 @@
   }
 
   &:focus#{$suffix} #{$target} {
-    border-color: tokens.get(tokens.$blue, 0) !important;
-    box-shadow: 0 0 0.25em 0.0625em tokens.get(tokens.$blue, 2);
-    outline-color: transparent;
-    outline-style: solid;
+    @include add-outline(tokens.$grey-1000, $offset);
+
+    &:active {
+      outline-offset: 0px;
+    }
   }
```

Comment:
we need a :focus:not(:focus-visible) for mouse users, i guess it should be no outline in that case

Reply:
There is on line 34 :)